lxl-web/src/routes/(app)/[[lang=lang]]/(find)/Pagination.svelte

```diff
@@ -95,3 +95,9 @@
 		</ul>
 	</nav>
 {/if}
+
+<style lang="postcss">
+	nav li > a {
+		@apply mobile-only:h-8 mobile-only:px-2.5;
```

Comment:
I think the buttons should be minimum size of 44px to adhere to: https://www.w3.org/WAI/WCAG21/Understanding/target-size.html.

I also prefer showing as many pages as possible (maybe with an upper limit of 10 instead of the current 3 in `sequenceSize`) so I would propose adding:
`
	nav li > a {
		@apply min-h-11 min-w-11 !px-2;
	}
` together with `flex-wrap` + `overflow-hidden` on a wrapping container around the pagination items (not the first and last as they should always be visible).

Reply:
Makes sense. Not really sure how you mean by the flex-wrap thing though, surely not this?

<img width="500" alt="Skärmavbild 2024-05-30 kl  14 13 43" src="https://github.com/libris/lxlviewer/assets/22232928/2d80e50d-11e5-4183-9e08-f0ee744db25c">

Reply:
You mean wrapping and hiding the middle (sequence) section? The current page would sometimes be in the hidden part...

Reply:
Ah, my proposed solution is probably not complete and needs some modification.

But in my head it would make sense to always keep the first, last and current (missed that one) pages visible. The number of items in the sequence around the current/first/last pages should be dynamic depending on the screen size (with an upper limit). The flex-wrap needs to be combined with overflow hidden to prevent items, and probably some containers.

It can be saved for a later PR if it requires too much reworking.

Reply:
Maybe it can be done in some clever way using pure CSS, but it eludes me atm. For example, the truncation (...) will have to appear when there is overflowing content.

Sort of reverted to the way it was, i.e showing the 'fixed' buttons for small screens. Looks better than before though, + bugs fixed. Should we settle for the time being?